actors.md

```diff
@@ -323,19 +318,10 @@ type StorageMiner struct {
 	// to this miner
 	PeerID peer.ID
 
-	// PledgeBytes is the amount of space being offered by this miner to the network
-	PledgeBytes BytesAmount
-
 	// SectorSize is the amount of space in each sector committed to the network
 	// by this miner.
 	SectorSize BytesAmount
 
-	// Collateral is locked up filecoin the miner has available to commit to storage.
-	// When miners commit new sectors, tokens are moved from here to 'ActiveCollateral'
-	// The sum of collateral here and in activecollateral should equal the required amount
-	// for the size of the miners pledge.
-	Collateral TokenAmount
```

Comment:
Note: The `DePledge` method [checks the `amt` against the `Collateral` field's value](https://github.com/filecoin-project/specs/pull/312/files#diff-8cd5872b87e5b64500c99ea50b3a9241R724). I think you should check `amt` against `vm.MyBalance() - miner.ActiveCollateral` instead.